src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Prepare an Azure account to run Giant Swarm guest clusters"
+description = "This guide will walk you through all necessary steps to set up an Azure account with approriate role definition and service principal for operating Giant Swarm guest clusters."
+date = "2018-07-18"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+
+# Prepare an Azure account to run Giant Swarm guest clusters
+
+In a Giant Swarm installation the guest clusters (the clusters running your Kubernetes workloads) can run in a separate Azure subscription (or account) from the host cluster. This gives great flexibility depending on the requirements and the usage scenario. For example, it allows the host cluster to be running in an Azure subscription owned by Giant Swarm, while guest clusters operate in different Azure subscriptions each, depending on a customer’s department using them.
+Giant Swarm operate guest clusters using a service called `azure-operator` which runs on the host cluster.
+
+## Overview
+
+In order to run Giant Swarm guest clusters, an Azure subscription need the following elements :
+
+- Role definition: set of permission to operate guest clusters in the Azure subscription.
+- Service principal: an identity (bound to the previoulsy defined role definition) to access the Azure subscription.
```

Comment:
`set of permission` is not clear to me. What something like `a set of permissions` or `setting the permission`